.github/workflows/examples-java-push.yml

```diff
@@ -65,20 +65,32 @@ jobs:
     environment: sandbox
     runs-on: ubuntu-18.04
     needs: test
+    env:
+      CF_API: "https://api.london.cloud.service.gov.uk"
+      CF_ORG: "cabinet-office-cddo-dsa-apis"
+      CF_SPACE: "sandbox"
     if: github.ref == 'refs/heads/main'
     steps:
-      - uses: actions/download-artifact@v2
+      - uses: actions/download-artifact@f023be2c48cc18debc3bacd34cb396e0295e2869
         with:
           name: examples-java-libs
-      - uses: actions/download-artifact@v2
+      - uses: actions/download-artifact@f023be2c48cc18debc3bacd34cb396e0295e2869
         with:
           name: spring-boot-manifest
+      - name: Install the CF CLI
+        run: |
+          wget -q -O cf.tar.gz "https://packages.cloudfoundry.org/stable?release=linux64-binary&version=7.4.0&source=github-rel"
```

Comment:
it would be good to see if there's any other way of pulling this that may allow us to Dependabot it?

Reply:
there might be, will have a look

Reply:
had a look but can't see a nice way to do that unfortunately